docs/mds/ARCHITECTURE_DOCUMENT.md

```diff
@@ -126,38 +127,37 @@ A cada sprint do projeto será definido a utilização ou não de cada component
 |[reportlab](https://pypi.org/project/reportlab/)|O Toolkit do ReportLab. Uma biblioteca Python de código aberto para gerar PDFs e gráficos.|Não|Foi recomendado na documentação do [Django](https://docs.djangoproject.com/pt-br/2.0/howto/outputting-pdf/), utilizar essa biblioteca para gerar os PDFs e gráficos do projeto. Porém a equipe achou uma maneira alternativa de faze-lo|
 |[django-easy-pdf](https://github.com/nigma/django-easy-pdf)|Visualização de PDF de uma maneira fácil|Não|A documentação do Django aconselha utilizar o ReportLab.|
-|[WeasyPrint](https://github.com/Kozea/WeasyPrint)|O WeasyPrint é um mecanismo de renderização visual para HTML e CSS que pode exportar para PDF. Destina-se a suportar os padrões da web para impressão. O WeasyPrint é um software gratuito disponibilizado sob uma licença BSD.|Não|Foi escolhido pela equipe o componente django-easy-pdf|
-|[sdcurve](https://github.com/Dr-Down/sdcurve)|Microsserviço de curva de crescimento, altura, peso, EMC e perímetro encefálico para crianças com Síndrome de Down|Sim|Requisitos do cliente.|
-|[Google Charts](https://developers.google.com/chart/)|API de plotagem de gráficos da google|Sim|Utilizado para plotar os gráficos de crescimento das crianças com Síndrome de Down|
+|[WeasyPrint](https://github.com/Kozea/WeasyPrint)|O WeasyPrint é um mecanismo de renderização visual para HTML e CSS que pode exportar para PDF. Destina-se a suportar os padrões da web para impressão. O WeasyPrint é um software gratuito disponibilizado sob uma licença BSD.|Não|Foi escolhido pela equipe o componente django-easy-pdf.|
+|[Google Charts](https://developers.google.com/chart/)|API de plotagem de gráficos da Google|Sim|Utilizado para plotar os gráficos de crescimento das crianças com Síndrome de Down.|
 
 #### Localização
 
 |Aplicação|Descrição da aplicação|Foi utilizado?|Motivo da utilização ou não|
 |---------|----------------------|:------------:|---------------------------|
-|[GoogleMapsAPI](https://developers.google.com/places/web-service/?hl=pt-br)|API do Google Maps com informações sobre milhões de locais| Sim |Facil aplicação e atende as necessidades.|
-|[google-maps-services-python](https://github.com/googlemaps/google-maps-services-python)|Componente python para utilização da API do google maps| Não | Não será ultilizado porque a API do google atendeu nossas necessidades e foi recomendada pela Gerência de projetos.|
+|[GoogleMapsAPI](https://developers.google.com/places/web-service/?hl=pt-br)|API do Google Maps com informações sobre milhões de locais.| Sim |Fácil aplicação e atende as necessidades.|
+|[google-maps-services-python](https://github.com/googlemaps/google-maps-services-python)|Componente python para utilização da API do google maps.| Não | Não será ultilizado porque a API do google atendeu nossas necessidades e foi recomendada pela gerência de projeto.|
```

Comment:
troca aqui o "e for recomendada pela gerência de projeto." por "e a equipe decidiu por manter o uso apenas na API."